source/hints.cpp

```diff
@@ -501,6 +501,26 @@ static Text BuildDungeonRewardText(ItemID itemID, const ItemKey itemKey) {
   return Text()+ITEM_OBTAINED(itemID)+"#"+GetHintRegion(Location(location)->GetParentRegionKey())->GetHint().GetText()+"#...^";
 }
 
+static Text BuildDoorOfTimeText() {
+  std::string itemObtained;
+  Text doorOfTimeText;
+
+  if (OpenDoorOfTime.Is(OPENDOOROFTIME_OPEN)) {
+    itemObtained = ITEM_OBTAINED(ITEM_SWORD_MASTER);
```

Comment:
I don't know if there's a way to remove the spacing for the image; When I kept `itemObtained` empty it used the previous image, and when I set it to `ITEM_NONE` it kept the spacing with no image. So I set it to the Master Sword as it is the path to the future.

Reply:
what if in that case you just return `Text() + doorOfTimeText` without the itemObtained?

Reply:
Then it still reuses the previous image like so:
![image](https://user-images.githubusercontent.com/5352197/127096609-fa5ddd2b-3b2e-45ba-93e1-d1eda23e368c.png)
![image](https://user-images.githubusercontent.com/5352197/127096579-f94f390c-ce50-4c9b-bec5-da74ae31809d.png)

Reply:
gotcha. then master sword image sounds good for now. If there is ever some time of "shuffle master sword" in the future, we can come back to it